src/middlewared/middlewared/plugins/kubernetes_linux/update.py

```diff
@@ -106,8 +106,16 @@ async def do_update(self, job, data):
 
         `node_ip` is the IP address which the kubernetes cluster will assign to the TrueNAS node. It defaults to
         0.0.0.0 and the cluster in this case will automatically manage which IP address to use for managing traffic
-        for default NAT network. If it is desired that traffic uses a certain interface / ip address, that IP address
-        can be specified and the NAT network will use related IP address and it's routes to manage the traffic.
+        for default NAT network.
+
+        By default kubernetes pods will be using default gateway of the system for outward traffic. This might
+        not be desirable for certain users who want to separate NAT traffic over a specific interface / route. System
+        will create a L3 network which will be routing the traffic towards default gateway for NAT.
+
+        If users want to restrict traffic over a certain gateway / interface, they can specify a default route
```

Comment:
I needed to read it two times before realising this would also be the way to deal with pushing k8s traffic through a vlan. Maybe clear this up a bit?
```suggestion
        If users want to restrict traffic over a certain gateway / interface / vlan, they can specify a default route
```

Reply:
Thank you @Ornias1993 for pointing it out however I am not sure about it because vlan is an interface ? Let's ask @freenas/docs-team for review so they can recommend what would be clearer and we can adjust accordingly.

Reply:
> vlan is an interface

I agree, so I got what you meant in the end... But if this is user facing, some users might not fully grasp the "virtual interface = still an interface" thingy...

Nice work this week though 👍